dirsrvtests/tests/suites/plugins/accpol_test.py

```diff
@@ -407,6 +407,142 @@ def test_glremv_lastlogin(topology_st, accpol_global):
+    for i in range(LOGIN_HIST_NUM_BINDS_SEVEN):
+        userconn = user.bind(USER_PW)
+        time.sleep(1)
+        userconn.unbind()
+
+    # Verify lastLoginTimeHistory attribute returns no more than lastLoginHistorySize entries in ascending order
+    entries = inst.search_s(USER_DN, ldap.SCOPE_SUBTREE, "(objectclass=*)", ['lastLoginHistory'])
+    entry_idx = 1
+    asc_order = 0
+    while entry_idx < len(entries[0].getValues('lastLoginHistory')):
+        if (entries[0].getValues('lastloginhistory')[entry_idx].decode() > entries[0].getValues('lastloginhistory')[entry_idx - 1].decode()):
+            asc_order = 1
+        entry_idx += 1
+    assert entry_idx == LOGIN_HIST_SIZE_FIVE
+    assert asc_order
+
+    # Reduce the lastLoginHistorySize to LOGIN_HIST_SIZE_TWO
+    try:
+        ap_config = ap_configs.create(properties={'cn': 'config', 'lastLoginHistorySize': str(LOGIN_HIST_SIZE_TWO)})
+    except ldap.ALREADY_EXISTS:
+        ap_config = ap_configs.get('config')
+        ap_config.replace('lastLoginHistorySize', str(LOGIN_HIST_SIZE_TWO))
+
+    # Bind as test user more times than lastLoginHistorySize
+    for i in range(LOGIN_HIST_NUM_BINDS_SEVEN):
+        userconn = user.bind(USER_PW)
+        time.sleep(1)
+        userconn.unbind()
+
+    # Verify lastLoginTimeHistory attribute returns no more than lastLoginHistorySize entries in ascending order
+    entries = inst.search_s(USER_DN, ldap.SCOPE_SUBTREE, "(objectclass=*)", ['lastLoginHistory'])
+    entry_idx = 1
+    asc_order = 0
+    while entry_idx < len(entries[0].getValues('lastLoginHistory')):
+        if (entries[0].getValues('lastloginhistory')[entry_idx].decode() > entries[0].getValues('lastloginhistory')[entry_idx - 1].decode()):
+            asc_order = 1
+        entry_idx += 1
+    assert entry_idx == LOGIN_HIST_SIZE_TWO
+    assert asc_order
+
+    # Increase the lastLoginHistorySize to LOGIN_HIST_SIZE_FIVE
+    try:
+        ap_config = ap_configs.create(properties={'cn': 'config', 'lastLoginHistorySize': str(LOGIN_HIST_SIZE_FIVE)})
+    except ldap.ALREADY_EXISTS:
+        ap_config = ap_configs.get('config')
+        ap_config.replace('lastLoginHistorySize', str(LOGIN_HIST_SIZE_FIVE))
+
+    # Bind as test user more times than lastLoginHistorySize
+    for i in range(LOGIN_HIST_NUM_BINDS_SEVEN):
+        userconn = user.bind(USER_PW)
+        time.sleep(1)
+        userconn.unbind()
+
+    # Verify lastLoginTimeHistory attribute returns no more than lastLoginHistorySize entries in ascending order
+    entries = inst.search_s(USER_DN, ldap.SCOPE_SUBTREE, "(objectclass=*)", ['lastLoginHistory'])
+    entry_idx = 1
+    asc_order = 0
+    while entry_idx < len(entries[0].getValues('lastLoginHistory')):
+        if (entries[0].getValues('lastloginhistory')[entry_idx].decode() > entries[0].getValues('lastloginhistory')[entry_idx - 1].decode()):
+            asc_order = 1
+        entry_idx += 1
+    assert entry_idx == LOGIN_HIST_SIZE_FIVE
+    assert asc_order
```

Comment:
And I think it can be rewritten with something like this? (and also encapsulated in a separate func))

```
    entries = inst.search_s(user_dn, ldap.SCOPE_SUBTREE, "(objectclass=*)", ['lastLoginHistory'])
    decoded_values = [entry.decode() for entry in entries[0].getValues('lastLoginHistory')]
    ascending_order = all(decoded_values[i] <= decoded_values[i + 1] for i in range(len(decoded_values) - 1))
    assert len(decoded_values) == login_hist_size
    assert ascending_order
```